src/config/chainEndpoints.ts

```diff
@@ -141,16 +141,14 @@ export const getProviderIndex = (chain: ASTAR_CHAIN) => {
   }
 };
 
-export const checkIsEnableIndividualClaim = async (api: ApiPromise): Promise<boolean> => {
+export const checkIsEnableNominationTransfer = async (api: ApiPromise): Promise<boolean> => {
```

Comment:
Since there is already compound reward check method. Maybe to create a single utility method which will check if any feature given as parameter is supported by a node.